### PR TITLE
Notify users of all ports in use by access keys

### PR DESCRIPTION
**Pull request by JonathanDCohen**
The old behavior was to use the port of the first access key, but this can miss many ports in use by the others.

New output looks like

`- Access key ports [ 25356,  12345,  60000,  21274,  6675,  29941,  55826,  38810,  5604,  5568,  43112 ] for TCP and UDP`

### CODE REVIEW

**Comment by fortuna** [comment]
I'm confused. In what situation will the install script have more than one access key?

**Comment by JonathanDCohen** [comment]
When you're re-installing the server manager, if you have access keys in the repo which use more than one port, then the output from the script is incorrect, as it only chooses the first key.

File: src/server_manager/install_scripts/install_server.sh [hunk 1]
```diff
@@ -352,7 +353,7 @@ $FIREWALL_STATUS
 
 Make sure to open the following ports on your firewall, router or cloud provider:
 - Management port ${API_PORT}, for TCP
-- Access key port ${ACCESS_KEY_PORT}, for TCP and UDP
+- Access key ports ${ACCESS_KEY_PORTS} for TCP and UDP
```
Comment (fortuna) [inline src/server_manager/install_scripts/install_server.sh]: ports -> port(s)

File: src/server_manager/install_scripts/install_server.sh [hunk 2]
```diff
@@ -330,18 +330,19 @@ function add_api_url_to_config() {
 }
 
 function check_firewall() {
-  local readonly ACCESS_KEY_PORT=$(curl --insecure -s ${LOCAL_API_URL}/access-keys | 
+  local readonly ACCESS_KEY_PORTS=$(curl --insecure -s ${LOCAL_API_URL}/access-keys |
```
Comment (fortuna) [inline src/server_manager/install_scripts/install_server.sh]: Can you make the output not have the surrounding brackets?

**Pull request closed without merging**